assets/ExpressionMetadata.json

```diff
@@ -163,6 +163,13 @@
             "minimumArguments": 1,
             "maximumArguments": 1
         },
```

Comment:
I assume you didn't mean to add all the `LanguageServerBin` files...

Reply:
Fixed